packages/editor/src/components/parts/condition/ConditionTable.tsx

```diff
@@ -69,30 +76,56 @@ const ConditionTable = ({ data, onChange }: { data: Condition[]; onChange: (chan
     }
   });
 
+  const showDeleteAction =
+    table.getSelectedRowModel().rows.length > 0 && !table.getRowModel().rowsById[Object.keys(rowSelection)[0]].original.target;
+
+  const tableActions = showDeleteAction
+    ? [
+        {
+          label: 'Remove row',
+          icon: IvyIcons.Trash,
+          action: () => removeRow(table.getRowModel().rowsById[Object.keys(rowSelection)[0]].index)
+        }
+      ]
+    : [];
```

Comment:
not quite sure, the old impl only showed a delete button for rows that are unknown. if this logic here does this, ignore this comment :)

Reply:
Yes with the "showDeleteAction" const this logic enables the same behavior as before 👍